### PR TITLE
update notes on vector layers and agol group

### PR DESCRIPTION
**Pull request by pmacMaps**
Here's an attempt to explain the link between the AGOL vector basemap group and the vector layer.

### CODE REVIEW

File: src/pages/api-reference/layers/vector-layer.md [hunk 1]
```diff
@@ -27,6 +27,17 @@ Extends [`L.Layer`](http://leafletjs.com/reference-{{siteData.latest_leaflet}}.h
...
+##### Esri Vector Basemaps Group
+
+Esri has developed many vector basemaps.  A complete listing of these is available on [ArcGIS Online](https://www.arcgis.com/home/group.html?id=30de8da907d240a0bccd5ad3ff25ef4a&view=list&start=1&focus=layers#content). Many of these basemaps provide support for local languages at large scales.  These have "Local Language" included in the item name. 
+
+The basemaps provide a mix of content, and can be grouped accordingly:
+- Map features and labels
+- Only map features
+- Only labels
+
+It may be possible to mix standard [basemaps](https://esri.github.io/esri-leaflet/api-reference/layers/basemap-layer.html) with label-only vector basemaps.
```
Comment (jgravois) [inline src/pages/api-reference/layers/vector-layer.md]: unless we feel _really_ solid about a suggestion, it probably shouldn't be officially documented.

File: src/pages/api-reference/layers/vector-layer.md [hunk 2]
```diff
@@ -27,6 +27,17 @@ Extends [`L.Layer`](http://leafletjs.com/reference-{{siteData.latest_leaflet}}.h
...
+
 ##### Custom Vector Tile Services
 
 Whether you've made a small tweak to one of Esri's hosted vector basemaps or published your own vector tile service from scratch you can draw the layer in Esri Leaflet by referencing the item's [`id`](http://www.arcgis.com/home/item.html?id=bd505ce3efff479bb4e87b182f180159).
```
Comment (jgravois) [inline src/pages/api-reference/layers/vector-layer.md]: it'd be useful to add a link to the vector tile style editor here.

https://www.esri.com/arcgis-blog/products/developers/mapping/design-custom-basemaps-with-the-new-arcgis-vector-tile-style-editor/

File: src/pages/api-reference/layers/vector-layer.md [hunk 3]
```diff
@@ -50,4 +61,8 @@ Whether you've made a small tweak to one of Esri's hosted vector basemaps or pub
 ```js
 var map = L.map('map').setView([37.75,-122.45], 12);
 L.esri.Vector.layer('bd505ce3efff479bb4e87b182f180159').addTo(map);
+
+// Item page for Light Gray Canvas Reference (Local Language) is https://www.arcgis.com/home/item.html?id=3ffec1551cd14606a286622c634b0bb4
+// You can pass in the item id to add this vector basemap
+L.esri.Vector.layer('3ffec1551cd14606a286622c634b0bb4').addTo(map);
```
Comment (jgravois) [inline src/pages/api-reference/layers/vector-layer.md]: it'd be better to share a 'self contained' basemap.

Comment (pmacMaps) [reply src/pages/api-reference/layers/vector-layer.md]: So, would it be better to share the link to the "map" items within the [basemaps group](https://www.arcgis.com/home/group.html?id=30de8da907d240a0bccd5ad3ff25ef4a&view=list&start=1&focus=maps#content)?

And then also to reference a web map item id in the example?

Comment (pmacMaps) [reply src/pages/api-reference/layers/vector-layer.md]: I think you meant a basemap that contains features and labels.  Had a hell of a day trying to rebuild a Drupal development site.

Comment (jgravois) [reply src/pages/api-reference/layers/vector-layer.md]: sorry, i mean vector tile item that already has labels (streets, topo, nav, etc.)

**Comment by pmacMaps** [comment]
Any other changes you want on this?

**Comment by jgravois** [comment]
nope. sorry!

i turned off notifications when folks push commits onto open pull requests.

**Pull request merged**